dbHelpers.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const db = require('./dbConfig');
+
+module.exports = {
```

Comment:
Placing your exports on the top here will work because of function declaration hoisting. I've mostly seen it placed on the bottom of files, but at the end of the day it will depend on your team's preferences, but as long as you're consistent with placement, and you know why this works you're good 👍  
You could also bundle up the whole thing if the file starts to get long
```javascript
module.exports = {  
    helperName: function() {}
    helperName: function() {}
}
```